AllASPFuncs.py

```diff
@@ -1292,3 +1307,34 @@ def banner(text):
     length = len(text) + 8
     message = "\n" + "#" * length +'\n'+'#   ' + text + '   # \n'+ "#" * length
     Lager.debug(message)
+
+
+def prep_data_for_fit(images, err, sn_matrix, wgt_matrix):
+    '''
+    This function takes the data from the images and puts it into the form such
+    that we can analytically solve for the best fit using linear algebra.
```

Comment:
Write out inputs and outputs.  Because the purpose of this function is explicitly to transfer a set of inputs into the same content but in a different structure, clearly explaining the expected form of the input and output is a key part of understanding the function.